dist/typescript-json-schema.js

```diff
@@ -1,262 +1,350 @@
-            "pattern", "minItems", "maxItems", "uniqueItems", "default",
-            "additionalProperties", "enum"];
-        JsonSchemaGenerator.annotedValidationKeywordPattern = /@[a-z.-]+\s*[^@]+/gi;
-        return JsonSchemaGenerator;
-    })();
-    function generateSchema(compileFiles, fullTypeName) {
-        var options = { noEmit: true, emitDecoratorMetadata: true, experimentalDecorators: true, target: 1, module: 1 };
-        var program = ts.createProgram(compileFiles, options);
-        var tc = program.getTypeChecker();
-        var diagnostics = program.getGlobalDiagnostics().concat(program.getDeclarationDiagnostics(), program.getSemanticDiagnostics());
-        if (diagnostics.length == 0) {
-            var allSymbols = {};
-            var inheritingTypes = {};
-            program.getSourceFiles().forEach(function (sourceFile) {
-                function inspect(node, tc) {
-                    if (node.kind == 214 || node.kind == 215) {
-                        var nodeType = tc.getTypeAtLocation(node);
-                        var fullName = tc.typeToString(nodeType, undefined, 128);
-                        allSymbols[fullName] = nodeType;
-                        nodeType.getBaseTypes().forEach(function (baseType) {
-                            var baseName = tc.typeToString(baseType, undefined, 128);
-                            if (!inheritingTypes[baseName]) {
-                                inheritingTypes[baseName] = [];
-                            }
-                            inheritingTypes[baseName].push(fullName);
-                        });
-                    }
-                    else {
-                        ts.forEachChild(node, function (node) { return inspect(node, tc); });
-                    }
-                }
-                inspect(sourceFile, tc);
-            });
-            var useRef = true;
-            var generator = new JsonSchemaGenerator(allSymbols, inheritingTypes, tc, useRef);
-            var definition = generator.getClassDefinitionByName(fullTypeName);
-            return definition;
-        }
-        else {
-            diagnostics.forEach(function (diagnostic) { return console.warn(diagnostic.messageText + " " + diagnostic.file.fileName + " " + diagnostic.start); });
-        }
-    }
-    TJS.generateSchema = generateSchema;
-    function exec(filePattern, fullTypeName) {
-        var files = glob.sync(filePattern);
-        var definition = TJS.generateSchema(files, fullTypeName);
-        console.log(JSON.stringify(definition, null, 4));
-    }
-    TJS.exec = exec;
-})(TJS = exports.TJS || (exports.TJS = {}));
-if (typeof window === "undefined" && require.main === module) {
-    if (process.argv[3]) {
-        TJS.exec(process.argv[2], process.argv[3]);
-    }
-    else {
-        console.log("Usage: node typescript-json-schema.js <path-to-typescript-files> <type>\n");
-    }
-}
+var ts = require("typescript");
```

Comment:
It's hard to see what happened. I believe there is some git option to ignore line endings. Maybe this helps https://help.github.com/articles/dealing-with-line-endings/